config/ignition_fortress_ubuntu_jammy.yaml

```
@@ -0,0 +1,17 @@
+name: ignition_fortress_ubuntu_jammy
+method: http://packages.osrfoundation.org/gazebo/ubuntu-stable
+suites: [jammy]
+component: main
+architectures: [amd64, i386, armhf, arm64, source]
```

```suggestion
architectures: [amd64, armhf, arm64, source]
```
Jammy does not support i386